gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -61,7 +65,7 @@ public static boolean isPointInTriangle (Vector3 point, Vector3 t1, Vector3 t2,
 		return true;
 	}
 
-	/** Returns true if the given point is inside the triangle. */
+	/** Returns {@code true} if the given point is inside the triangle. */
```

Comment:
I don't see this as being important. It is clear what it means either way. `true` is slightly prettier in javadocs but uglier in the code. I think it is much more common to read the code than to read the javadocs, so I would vote against using @code in most places.

Reply:
No, it is not required.